src/main/java/core/basesyntax/Reader.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class Reader {
+    public List<String> readCsvFileWithData(Path pathFile) {
+        List<String> result;
+        try {
+            result = Files.readAllLines(pathFile);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file " + pathFile, e);
+        }
```

Comment:
Good use of try-with-resources to handle potential `IOException`. However, the `result` variable can be declared inside the try block since it's not used outside of it.